src/scss/components/_icons.scss

```diff
@@ -0,0 +1,87 @@
+/*
+   Icons
+   ========================================================================== */
+
+
+[class^='c-icon '],
+[class^='c-icon-'],
+[class*=' c-icon '],
```

Comment:
Wouldn't 

```css
[class*='c-icon']
```

on its own be sufficient?

[Tested in this CodePen](https://codepen.io/DamianMullins/pen/mMRzbX?editors=1100).